accounts/admin.py

```diff
@@ -17,11 +16,9 @@
 import django.core.urlresolvers
 import django.views.decorators.debug
 from django.utils.translation import ugettext_lazy as _
-
 import signals
 import models
 
-
```

Comment:
@mohannadsh This should be here, there should be two empty lines between imports and next definitions.